fluentd/configs.d/openshift/filter-post-genuuid.conf

```diff
@@ -0,0 +1,6 @@
+# generate uuids at source mitigate creating duplicate records
+# requires fluent-plugin-elasticsearch
+<filter **>
+  @type elasticsearch_genid
+  hash_id_key _openshift_es_genid
```

Comment:
this should be `openshift_es_genid` and it should be part of the official schema, part of the published api
This field will be useful for users to be able to correlate their own data with the data in Elasticsearch, and will be very useful in alerting applications (to alert on ES data) and for message queue applications.